modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/utils/showSidebar.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {render} from 'frontend-js-react-web';
+
+import SidebarPanel from '../SidebarPanel';
+import SidebarPanelInfoView from '../components/SidebarPanelInfoView';
+import SidebarPanelMetricsView from '../components/SidebarPanelMetricsView';
+
+const actions = {
+	showInfo(fetchURL, namespace) {
+		showSidebar({View: SidebarPanelInfoView, fetchURL, namespace});
+	},
+	showMetrics(fetchURL, namespace) {
+		showSidebar({View: SidebarPanelMetricsView, fetchURL, namespace});
+	},
+};
+
+const showSidebar = ({View, fetchURL, namespace}) => {
+	const _getSidebarPanel = () => {
+		return Liferay.component(`${namespace}sidebar`);
+	};
+
+	const _setSidebarPanel = (element) => {
```

Comment:
This could probably just be passed in line maybe.

Reply:
Yep again!